discovery-provider/src/api/v1/tracks.py

```diff
@@ -602,6 +603,37 @@ def get(self, track_id):
         stems = list(map(stem_from_track, stems))
         return success_response(stems)
 
+track_remixables_route_parser = reqparse.RequestParser()
+track_remixables_route_parser.add_argument("user_id", required=False)
+track_remixables_route_parser.add_argument("limit", required=False, type=int)
+track_remixables_route_parser.add_argument("with_users", required=False, type=bool)
+@ns.route("/remixables")
+class RemixableTracks(Resource):
+    @record_metrics
+    @ns.doc(
```

Comment:
as discussed, this should be a full_ns route and we can open the non full one later with other explore endpoints. in the mean time, the /full will suffice to give us AudiusAPIClient  https://github.com/AudiusProject/audius-client/blob/6953106a16069a84726e57755965166ac98b2ab1/src/services/audius-api-client/AudiusAPIClient.ts#L39